### DISCUSSION TITLE
Push Notifications

### DISCUSSION THREAD

**Discussion by Moosems** [community/community#20222] [Mobile]
Can you add an option to give notifications for any action that would give a notification on the website. For example: if I am in a thread for a discussion I would like to get push notifications for any comments on it even if I am not directly mentioned. If this exists I was unaware because on the push notifications selection page it didn't have this option

Picture of notification options:

<img src="https://user-images.githubusercontent.com/95927277/178091726-19222890-0814-49e4-972c-954557b6e807.png">

**Comment by alexwholland** [comment]
Does the Subscribe button on the right of each thread not work?
![image](https://user-images.githubusercontent.com/64433812/178091191-4c41357d-da5e-4ad2-91ae-6640c435fde6.png)

**Reply by Moosems** [reply]
I didn't even get a notification for this and I am subscribed so apparently not

**Reply by Moosems** [reply]
![image](https://user-images.githubusercontent.com/95927277/178091726-19222890-0814-49e4-972c-954557b6e807.png)

**Comment by ferhatiltas** [comment]
go to settings and turn on notifications

**Comment by Moosems** [comment]
@ferhatiltas They are on
